Description: Disable examples and remove related dev deps
Last-Update: 2022-12-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,2 +35,3 @@
 repository = "https://github.com/gyscos/zstd-rs"
+autoexamples = false
 
@@ -45,3 +46,3 @@
-[[example]]
-name = "train"
-required-features = ["zdict_builder"]
+#[[example]]
+#name = "train"
+#required-features = ["zdict_builder"]
@@ -54,3 +55,3 @@
-[dev-dependencies.clap]
-version = "4.0"
-features = ["derive"]
+#[dev-dependencies.clap]
+#version = "4.0"
+#features = ["derive"]
@@ -58,2 +59,2 @@
-[dev-dependencies.humansize]
-version = "2.0"
+#[dev-dependencies.humansize]
+#version = "2.0"
